### ISSUE TITLE
MBeans plugin doesn't repaint window

### ISSUE THREAD

**Issue by mihxil** [visualvm/visualvm.src#3]
The tabs on the right side start out totally gray. I need to do something which causes this window to repaint to view its content. Resizing it or so.
Before:
![screen shot 2016-10-13 at 21 29 01](https://cloud.githubusercontent.com/assets/429477/19363890/30fbf3de-918c-11e6-8815-5e92201cb73a.png)

After:
![screen shot 2016-10-13 at 21 31 03](https://cloud.githubusercontent.com/assets/429477/19363945/6212da3c-918c-11e6-8a24-2b93abcd13f7.png)

**Comment by kzadorozhny** [comment]
+1. Experiencing the same issue here.

```
Version: 
1.3.9 (Build 161004); platform 20161002-268c3bcd7a42

System: 
Mac OS X (10.12) , x86_64 64bit

Java: 

1.8.0_92; Java HotSpot(TM) 64-Bit Server VM (25.92-b14, mixed mode)

Vendor: 
Oracle Corporation, http://java.oracle.com/

Environment: 
UTF-8; en_US (visualvm)

Clusters: 
/Applications/VisualVM.app/Contents/platform
/Applications/VisualVM.app/Contents/visualvm
/Applications/VisualVM.app/Contents/profiler

```

**Comment by missedone** [comment]
have the same issue, please fix the annoying thing.
thanks

**Comment by winmain** [comment]
Have the same issue after updating VisualVM-MBeans plugin to 1.7

**Comment by thurka** [comment]
The fix was back-ported to VisualVM 1.3.9. New version 1.7.1 of MBeans plugin is now available on VisualVM 1.3.9 plugins center.
